Source/RMG-Core/Settings/Settings.cpp

```diff
@@ -547,6 +547,9 @@ static l_Setting get_setting(SettingsID settingId)
     case SettingsID::RomBrowser_ColumnSizes:
         setting = {SETTING_SECTION_ROMBROWSER, "ColumnSizes", std::vector<int>({-1, -1, -1})};
         break;
+    case SettingsID::RomBrowser_ColumnVisibility:
```

Comment:
Why not just call thix `RomBrowser_Columns`? It'd make sense for yor third step anyways, just have more columns but just having some hidden by default.

Reply:
Done, will include in next commit.